daprdocs/content/en/reference/components-reference/supported-pubsub/_index.md

```diff
@@ -26,6 +26,7 @@ Table captions:
 | [Hazelcast]({{< ref setup-hazelcast.md >}})           | Alpha  | v1 | 1.0 |
 | [MQTT]({{< ref setup-mqtt.md >}})                     | Alpha  | v1 | 1.0 |
 | [NATS Streaming]({{< ref setup-nats-streaming.md >}}) | Beta  | v1 | 1.0 |
+| [JetStream]({{< ref setup-jetstream.md >}}) | Beta  | v1 | 1.4 |
```

Comment:
All components start as Alpha, please change.

Reply:
Ah, whoops. I've changed this to Alpha. https://github.com/dapr/docs/pull/1751/commits/0d46589f18593d4b4395ccfcd88f27a569dc2ece